Tests/StreamChatTests/APIClient/Endpoints/Payloads/IdentifiableModel_Tests.swift

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChat
+@testable import StreamChatTestTools
+import XCTest
+
+final class IdentifiableModel_Tests: XCTestCase {
+    var database: DatabaseContainer!
+
+    private lazy var deviceDTO: DeviceDTO = {
+        let dto = DeviceDTO(context: database.viewContext)
+        dto.id = "DeviceDTO_123"
+        return dto
+    }()
+
+    override func setUp() {
+        super.setUp()
+        database = DatabaseContainer_Spy()
+    }
+
+    func test_ChannelDTO_conformsToIdentifiableModel() {
```

Comment:
Can we generalize these tests somehow?
1. Get all Models registered in the DB scheme
2. Iterate over them and do what's done in these tests

Reply:
I guess the answer to this was:
> I am in favor of making it a bit more readable than parametrizing tests. In the end, more human errors can happen if we do what you suggest.

Fair enough, I wanted to handle the case where we add a new DTO and forget to add a test for it in this file